### PR TITLE
Updated users on Cloud article to reflect the new UmbracoID invite flow

### PR DESCRIPTION
**Pull request by jonat123**

### CODE REVIEW

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 1]
```diff
@@ -23,7 +23,41 @@ Users can also be invited directly from the backoffice of your Umbraco Cloud pro
...
+## Invite User through the Umbraco backoffice
+
+As mentioned it is possible to invite new Users to your Umbraco Cloud project through the backoffice as you would on a normal installation of Umbraco.
+
+To invite a User you need to do the following:
+
+1. Go to the backoffice of your Umbraco Cloud project
+2. Go to the User section in the backoffice
+3. Invite new User to the project
+4. Enter name and email and add a User Group to assign access and permissions and optionaly enter a new message for the invitation
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:38]:
```suggestion
4. Enter name and email and add a User Group to assign access and permissions and optionally enter a new message for the invitation
```

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 2]
```diff
@@ -23,7 +23,41 @@ Users can also be invited directly from the backoffice of your Umbraco Cloud pro
...
+
+![Invite User](images/invite_user.png)
+
+Once the User have been invited they will receive an invitation for the project.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:42]:
```suggestion
Once the User has been invited they will receive an invitation for the project.
```

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 3]
```diff
@@ -23,7 +23,41 @@ Users can also be invited directly from the backoffice of your Umbraco Cloud pro
...
+
+If you already have a User on Umbraco Cloud you will get a invitation in the project portal under "Project Invites".
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:44]:
```suggestion
If the user being invited already has a User on Umbraco Cloud they will be able to see the invitation in the project portal under "Project Invites".
```

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 4]
```diff
@@ -23,7 +23,41 @@ Users can also be invited directly from the backoffice of your Umbraco Cloud pro
...
+
+If the User being invited does not have a User on Umbraco Cloud, they will receive a email to create one.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:46]:
```suggestion
If the User being invited does not have a User on Umbraco Cloud, they will receive an email asking them to create one.
```

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 5]
```diff
@@ -23,7 +23,41 @@ Users can also be invited directly from the backoffice of your Umbraco Cloud pro
...
+
+ ![New User Invitation](images/New_user.png)
+
+ Once the User have been created, it is now possible to login to the Umbraco Cloud portal.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:50]:
```suggestion
 Once the User has been created, it is now possible for them to login to the Umbraco Cloud portal.
```

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 6]
```diff
@@ -23,7 +23,41 @@ Users can also be invited directly from the backoffice of your Umbraco Cloud pro
...
+
+ From here they will have a pending invitation to the project they have been invited to.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:52]:
```suggestion
 From here they will be able to see a pending invitation to the project they have been invited to.
```

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 7]
```diff
@@ -23,7 +23,41 @@ Users can also be invited directly from the backoffice of your Umbraco Cloud pro
...
+
+ Once the invitation have been accepted, they can now access the project through the Umbraco Cloud portal and access their site from there.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:54]:
```suggestion
 Once the invitation has been accepted, they can now access the project through the Umbraco Cloud portal and access their site from there.
```

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 8]
```diff
@@ -23,7 +23,41 @@ Users can also be invited directly from the backoffice of your Umbraco Cloud pro
...
+
+  ![New User Invitation](images/project_overview.png)
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:56]:
```suggestion
  ![New User Invitation](images/Project_overview.png)
```

**Review by sofietoft — COMMENTED** [review]
Great additions about the new flow for accepting an invitation for the Umbraco backoffice on a Cloud project.

I've made a few suggestions here and there - mostly some grammar stuff.
And then there was also an issue with the final image.

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 9]
```diff
@@ -23,7 +23,41 @@ Users can also be invited directly from the backoffice of your Umbraco Cloud pro
...
+
+1. Go to the backoffice of your Umbraco Cloud project
+2. Go to the User section in the backoffice
+3. Invite new User to the project
+4. Enter name and email and add a User Group to assign access and permissions and optionally enter a new message for the invitation
+
+![Invite User](images/invite_user.png)
+
+Once the User have been invited they will receive an invitation for the project.
+
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:43]: Would it make sense to add an H3 (###) heading here, so it's clear that the next part is about how to accept an invite to the backoffice?

File: Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md [hunk 10]
```diff
@@ -23,7 +23,41 @@ Users can also be invited directly from the backoffice of your Umbraco Cloud pro
...
+ From here they will be able to see a pending invitation to the project they have been invited to.
+
+ Once the invitation has been accepted, they can now access the project through the Umbraco Cloud portal and access their site from there.
+
+  ![New User Invitation](images/Project_overview.png)
+
+:::note
+This only applies on project on version 8.9.1 and above.
+Users invited to projects on lower versions will not need an Umbraco Cloud User.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:60]: The version number here should be updated to 8.6.9 - correct? 🤔 

And also, I think it makes sense that a note like this goes to the top of the section, as it is something you would want to know before you start learning about the feature.

Comment (jonat123) [reply Umbraco-Cloud/Set-Up/Users-On-Cloud/index.md:60]: It makes perfect sense to move it up top, regarding the 8.6.9 as I understand it has not been implemented on those versions yet, and the projects already on 8.9.x will be upgraded on the 9th of February as it says on the status page, so maybe until it have been upgraded i can write something like "This only applies on new project on version 8.9.1 and above." then we can change it when it have been implemented :-)

**Pull request merged**